external-crates/move/crates/move-model-2/src/model.rs

```diff
@@ -0,0 +1,1019 @@
+// Copyright (c) The Move Contributors
```

Comment:
Apologies. I should have put the renaming as a separate PR, since now it is rather difficult to see what is different. The only thing "new" here are the spots with the const generics. If it becomes difficult to review. I can make separate PRs